admin/views/class-yoast-input-select.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class for generating a html select.
+ */
+class Yoast_Input_Select {
+
```

Comment:
Making a method to provide selected option cleans up the constructor and still keeps needed functionality.